### PR TITLE
Added 1.6 Upgrade notes to outline changes for etcd 3.

### PR DESCRIPTION
**Pull request by timothysc**
Fixes: #2172

/cc wojtek-t

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2763)

### CODE REVIEW

**Comment by wojtek-t** [comment]
@mml - FYI

This looks ok to me, but we need to extend this doc with the description how we recommend doing the upgrade. Though, we should probably merge this PR and do it in the next one (probably either @mml or myself).

File: docs/admin/upgrade-1-6.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+assignees:
+- mml
+title: Cluster Management Guide
+---
+
+* TOC
+{:toc}
+
+This document outlines the potentially distruptive changes that exist in the 1.6 release cycle.  Operators, administrators, and developers should
```
Comment (derekwaynecarr) [inline docs/admin/upgrade-1-6.md]: typo: disruptive

File: docs/admin/upgrade-1-6.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+continuity.  
+
+It is possible to maintain v2 compatibility mode while running etcd 3 for an interim period of time.  To do this, you will simply need to update an argument passed to your apiserver during 
+startup: 
+
+```
+$ kube-apiserver --storage-backend='etcd2' $(EXISTING_ARGS)
+``` 
+
+However, for longterm maintainance of the cluster, we recommend that the operator plan an outage window in order to perform a [v2->v3 data upgrade](https://coreos.com/etcd/docs/latest/upgrades/upgrade_3_0.html).
```
Comment (derekwaynecarr) [inline docs/admin/upgrade-1-6.md]: typo: maintenance

**Comment by timothysc** [comment]
> but we need to extend this doc with the description how we recommend doing the upgrade.

@wojtek-t  I explicitly linked to the coreos docs on that part, but we could have a second on GCE/GKE using the script-ery too.

**Comment by wojtek-t** [comment]
@timothysc - LGTM, though please fix travis - the failure seems related.

**Comment by timothysc** [comment]
@wojtek-t DONE.

**Comment by timothysc** [comment]
@wojtek-t It looks like this needs an explicit merge.

**Comment by wojtek-t** [comment]
@timothysc - oops sorry. 

Can you please rebase and ping me?

**Comment by timothysc** [comment]
@wojtek-t done.

**Comment by wojtek-t** [comment]
@timothysc - done :)

**Pull request merged**